corehq/apps/enterprise/enterprise.py

```diff
@@ -405,6 +422,94 @@ def _get_individual_export_rows(self, exports, export_line_counts):
         return rows
 
 
+class EnterpriseCommCareVersionReport(EnterpriseReport):
+    title = gettext_lazy('CommCare Version Compliance')
+    total_description = gettext_lazy('% of Mobile Workers on the Latest CommCare Version')
+
+    @property
+    def headers(self):
+        return [
+            _('Mobile Worker'),
+            _('Project Space'),
+            _('Latest Version Available at Submission'),
+            _('Version in Use'),
+        ]
+
+    @property
+    def rows(self):
+        config = CommCareBuildConfig.fetch()
+        partial_func = partial(self.rows_for_domain, config)
+        return _process_domains_in_parallel(partial_func, self.account.get_domains())
+
+    @property
+    def total(self):
+        total_mobile_workers = 0
+        total_up_to_date = 0
+        config = CommCareBuildConfig.fetch()
+
+        def total_for_domain(domain_name):
+            mobile_workers = get_mobile_user_count(domain_name, include_inactive=False)
+            outdated_users = len(self.rows_for_domain(domain_name, config))
+            return mobile_workers, outdated_users
+
+        results = _process_domains_in_parallel(total_for_domain, self.account.get_domains())
+
+        for domain_mobile_workers, outdated_users in results:
+            if domain_mobile_workers:
+                total_mobile_workers += domain_mobile_workers
+                total_up_to_date += domain_mobile_workers - outdated_users
+
+        return _format_percentage_for_enterprise_tile(total_up_to_date, total_mobile_workers)
+
+    def rows_for_domain(self, domain_name, config):
```

Comment:
Nit: I believe we refer to a domain object as `domain_obj` and the domain name as `domain`

Reply:
7599675a56c9820eb540ff0802cf236a9e549f4e